### DISCUSSION TITLE
Modal screen identifies desktop browser as mobile version

### DISCUSSION THREAD

**Discussion by berma83** [Budibase/budibase#2519] [Help]
Hello,

I don't know if this is a bug but modal screen identifies browser as mobile version.

Current Version: 0.9.117

<img src="https://user-images.githubusercontent.com/53205779/131759513-c5957077-ac7f-4749-aa57-59cabc709fc5.png" width="50%">

**Modal**
<img src="https://user-images.githubusercontent.com/53205779/131759456-28fd25ab-40ec-4ead-8fb6-34fd72f9de67.png" width="50%">

**Fullscreen**
<img src="https://user-images.githubusercontent.com/53205779/131759632-57acb93e-ff0a-4f1b-8f7b-585945ecd39a.png" width="50%">

**Comment by mjashanks** [comment]
I can reproduce this. However, I would say it is **not** a bug. Even though the modal is technically not "Mobile", it is Mobile **in size** - meaning that we can reformat it accordingly for the size of the modal.

I think this is ok. It might be an idea to be able to control the size of the Modal, which we cannot do right now, but has been suggested elsewhere.
